### DISCUSSION TITLE
Benefits Not Received

### DISCUSSION THREAD

**Discussion by lemunzzz62** [community/community#120141] [GitHub Education] [answered]
### Hi everyone,

My application was made April 15th, and verified soon after - on the same day. But I have yet to receive the benefits. I asked one of my friends who got his, and he said he got the benefits some 20 minutes after his initial verification. Did I do something wrong? Help please.

**Comment by Mriganka69** [comment]
**It sounds like you’re experiencing a delay in receiving benefits after your application and verification. There could be several reasons for this, and it doesn’t necessarily mean you did something wrong. Processing times can vary due to a number of factors, such as the volume of applications being handled, individual circumstances, or even system errors.**

**Reply by lemunzzz62** [reply]
What's the average wait time? Cus I feel like this is kinda long compared to how long it took for my application to get approved.

**Reply by lemunzzz62** [reply, marked as answer]
Nevermind, I just got them :D

**Comment by navinrmeena** [comment]
i got mail that my copilot is expire but i am a student .
<img width="1437" alt="Screenshot 2024-04-20 at 12 54 08 PM" src="https://github.com/community/community/assets/103177645/fc125424-268d-4c75-a9f3-74c3ae6a0671">
![Screenshot 2024-04-20 at 12 54 02 PM](https://github.com/community/community/assets/103177645/bed7edbd-8576-42a9-9092-3989bfc26894)
